vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow-integration-tests/src/test/java/com/vaadin/flow/component/gridpro/tests/BasicIT.java

```diff
@@ -145,6 +145,20 @@ public void cellEditStartedListenerCalledOnce() {
                         + "department=sales, city='City 1', employmentYear=2019}"));
     }
 
+    @Test
+    public void itemClickListenerListenerCalledOnce() {
+        GridTHTDElement cell = grid.getCell(0, 2);
+        cell.click();
+
+        assertCellEnterEditModeOnDoubleClick(0, 2, "vaadin-combo-box");
+        String eventsPanelText = getPanelText("events-panel");
+        Assert.assertEquals(1, eventsPanelText.split("ItemClicked").length - 1);
+        Assert.assertTrue(eventsPanelText
```

Comment:
```suggestion
        GridTHTDElement cell = grid.getCell(0, 2);
        cell.click();
        Assert.assertEquals(1, eventsPanelText.split("ItemClicked").length - 1);

        assertCellEnterEditModeOnDoubleClick(0, 2, "vaadin-combo-box");
        String eventsPanelText = getPanelText("events-panel");
        Assert.assertEquals(1, eventsPanelText.split("ItemClicked").length - 1);
        Assert.assertTrue(eventsPanelText
```

Reply:
That `assertCellEnterEditModeOnDoubleClick` line serves no purpose in the test at all. The helper method doesn't actually perform a double-click on the line but dispatches a "dblclick" event manually.

Removed the line altogether since it isn't related to the test case. An actual double-click on the row would dispatch 2 additional item click listener invocations.